cpg_workflows/targets.py

```diff
@@ -493,7 +513,7 @@ def __init__(
         sex: Sex | None = None,
         pedigree: Optional['PedigreeInfo'] = None,
         alignment_input_by_seq_type: dict[str, AlignmentInput] | None = None,
-        seq_by_type: dict[str, Sequence] | None = None,
+        assays: dict[str, Assay] | None = None,
```

Comment:
Is this keyed by external id, otherwise why a string?

Reply:
It's keyed by assay-type, which AFAIK is a string, I can define an enum for it though if that's better? It's meant to be the direct replacement of seq_by_type, so I can alternatively call it assays_by_type